### PR TITLE
Qingwei/show chart for statement

### PR DESCRIPTION
**Pull request by qingweibinary**
I move the ContractDetailsContainer (The Modal containing Chart) into workspace, and let statement and portfolio use the same instance. 

I named it singleton, which may or may not be appropriate, so please comment

### CODE REVIEW

File: src/_singletons/ContractDetailsSingleton.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import React, { PropTypes, Component } from 'react';
+import { connect } from 'react-redux';
+import Modal from '../containers/Modal';
+import ContractDetailsContainer from '../contract-details/ContractDetailsContainer';
+import ContractDetailsSingletonSelector from './ContractDetailsSingletonSelector';
+
+@connect(ContractDetailsSingletonSelector)
+export default class ContractDetailsSingleton extends Component {
```
Comment (borisyankov) [inline src/_singletons/ContractDetailsSingleton.js]: Possibly Container is better name than Singleton.
Singleton is describing a bit of its purpose, but not rally the pattern, as I can certainly create more than one of it ;)

**Comment by borisyankov** [comment]
Apart from the Singleton naming, I think its good ;)

File: src/_singletons/ContractDetailsSingleton.js (hunk 1 above)
Comment (qingweibinary) [inline src/_singletons/ContractDetailsSingleton.js]: OK, container already being used, I'll come up with another name.

And fix the test

**Comment by qingweibinary** [comment]
@borisyankov updated and test passed

**Pull request merged**